src/microbe_stage/organelle_components/MovementComponent.cs

```diff
@@ -38,7 +38,6 @@ public override void Update(float elapsed)
     protected override void CustomAttach()
     {
         force = CalculateForce(organelle.Position, Momentum);
-
```

Comment:
unnecessary line removal which puts unrelated stuff in the same code "block", which I dislike